data/kinds.yml

```diff
@@ -28,6 +28,7 @@
 - id: explanation
   name: Explanations
   description: ""
+  path: /explanations/
```

Comment:
This is fine for now, but I'm not sure about having "Explanations" on the landing page.

![image](https://user-images.githubusercontent.com/639336/89330356-f84ae700-d644-11ea-8e5f-02165aa1776c.png)

I didn't have this `kind` show up on the top page because I wasn't sure if it was something that makes sense there. When a user visits that page, I can imagine them looking for other kinds, but "Explanations"?

We might want to change it to "Concepts" later if we want it on the landing. At least, I think that's more clear what they'll find.

Reply:
do you wan't me to update that already? (not a problem)

Reply:
Don't you work on GitHub? I was going to do global find and replace because that's easier and safer.

It's not a huge issue, so we can leave it for now.

Reply:
> Don't you work on GitHub?

I work directly using firefowx, actually... x)

Reply:
Yeah, that's what I meant. I'll do the renaming later.

Reply:
I do not like the word "Explanations" either, but I believe this is exactly the kind of docs B4B mentioned sometimes before, and I thought could replace the "About" section from old wiki:

![image](https://user-images.githubusercontent.com/23709795/89387544-e891d800-d702-11ea-82cf-f5056fb10458.png)

Unless you would like to have it organized completely differently, I think it would be a good place to have some "articles" that explain some aspects in *descriptive* way, as opposed to references which are just collection of "technical" points. I would see it as something similar to HowTo or Tutorial, but not focused on a problem/goal to achieve, but on a concept or domain element. "Concepts" is nice, and I think it would be good to have it on main page, because if not, then how else could it be discovered?

Reply:
yeah, "concepts" seems to be the best candidate for this :+1:

Reply:
OK, I'll do global find replace to change it to Concepts after this is merged.